modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/js/utils/clean_suggestions_contributor_configuration.js

```diff
@@ -16,9 +16,10 @@ import parseJSONString from './parse_json_string';
 import {CONTRIBUTOR_TYPES} from './types/contributorTypes';
 
 /**
- * Filters out blueprint suggestion contributors in the fields array
- * if search experiences are not supported.
- * @param {Array} fields The list of fields.
+ * Filters out blueprint suggestion contributors in the
+ * `suggestionsContributorConfiguration` array if search experiences
+ * are not supported.
+ * @param {Array} suggestionsContributorConfiguration
```

Comment:
`suggestionsContributorConfiguration` is a string. Can you also add a description that this is a string of an array of objects?

Reply:
Oh my bad, didn't see that, thanks! I'll add it in.